bucket/rescuetime.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "2.14.5.3",
+    "description": "Automatic application time-tracking and productive advisor",
+    "homepage": "https://www.rescuetime.com",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.rescuetime.com/tos"
+    },
+    "url": "https://www.rescuetime.com/installers/RescueTimeInstaller_2.14.5.3.exe",
+    "hash": "abe2005aa5a2fa700ca3faf88c6ed36638448427c202af1a716094639601e0bd",
+    "innosetup": true,
+    "bin": "RescueTime.exe",
+    "shortcuts": [
+        [
+            "RescueTime.exe",
+            "RescueTime"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.rescuetime.com/updates/win_release_notes.html",
+        "regex": "<strong>([\\d.]+)</strong>"
+    },
+    "autoupdate": {
+        "url": "https://www.rescuetime.com/installers/RescueTimeInstaller_$version.exe"
+    },
+    "notes": [
```

Comment:
```suggestion
    "notes": "Consider installing extension for browser."
```

Reply:
That doesn't sound grammatically right to me...
`Consider installing the RescueTime extension for your browser.` sounds much better.

Reply:
Whatever works for you. Main change is string instead of array.

Reply:
There you go.